sdk/communication/azure-communication-phonenumbers/samples/get_phone_number_sample.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: phone_number_area_codes_sample.py
+DESCRIPTION:
+    This sample demonstrates how to get all area codes via a connection string, country code and phone plan id.
+USAGE:
+    python list_acquired_phone_numbers_sample.py
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING - The endpoint of your Azure Communication Service
+    2) AZURE_COMMUNICATION_SERVICE_PHONE_NUMBER - The phone number you want to get its information
+"""
+
+import os
+from azure.communication.phonenumbers import (
+    PhoneNumbersClient
+)
+
+connection_str = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
+phone_number = os.getenv("AZURE_COMMUNICATION_SERVICE_PHONE_NUMBER")
```

Comment:
For a sample - it would actually be more helpful to see what this value should look like